superset/db_engine_specs.py

```diff
@@ -873,11 +873,12 @@ def get_column_names(filepath):
         s3 = boto3.client('s3')
         location = os.path.join('s3a://', bucket_path, upload_prefix, table_name)
         s3.upload_file(
-            upload_path, 'airbnb-superset',
+            upload_path, bucket_path,
             os.path.join(upload_prefix, table_name, filename))
         sql = """CREATE EXTERNAL TABLE {table_name} ( {schema_definition} )
             ROW FORMAT DELIMITED FIELDS TERMINATED BY ',' STORED AS
-            TEXTFILE LOCATION '{location}'""".format(**locals())
+            TEXTFILE LOCATION '{location}'
+            tblproperties ('skip.header.line.count'='1')""".format(**locals())
```

Comment:
Note the `skip.header.line.count` property will only work in Presto for v0.199 or later per [here](https://prestodb.github.io/docs/current/release/release-0.199.html#hive-changes).